tests/clip_vertex_data_test.py

```diff
@@ -109,11 +112,42 @@ def test_clip_curve_data(tmp_path):
     with workspace.open():
         with Workspace(tmp_path / r"testClipPoints_copy2D.geoh5") as new_workspace:
             clipped_pts = curve.copy_from_extent(parent=new_workspace, extent=extent)
-            clipped_d = clipped_pts.get_data("VertexValues")[0]
-            clipped_c = clipped_pts.get_data("CellValues")[0]
-            assert clipped_pts.n_vertices == clippings.sum()
-            assert np.all(clipped_d.values == data[0].values[clippings])
-            assert len(clipped_c.values) == clipped_pts.n_cells
+            assert clipped_pts is None
+
+
+def test_clip_surface(tmp_path):
```

Comment:
**suggestion (test):** have a test for surface with `inverse=False`

Reply:
Done